zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template/parameters.rs

```diff
@@ -11,7 +11,6 @@ pub enum GetBlockTemplateRequestMode {
     Template,
 
     /// Indicates a request to validate block data.
-    /// Currently unsupported and will return an error.
```

Comment:
Please also remove the comment on the enum that says this, and search for similar comments.

Reply:
Fixed in [`8420b75`](https://github.com/ZcashFoundation/zebra/pull/5870/commits/8420b75013c1a05ea11a9799d1f87bd338c3c27c), I think all the comments to this effect included "unsupported" and searched for that.